examples/flight-management/backend/helpters/DatabaseManager.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+export default class DatabaseManager {
+  api = axios.create({ baseURL: 'http://localhost:3004' })
```

Comment:
I'd call this `dbAPI` instead. In the initial round of review, I thought you were creating an HTTP API but then realized you're just using json-server to interface with the db.json file. BTW, you might want to remove the json-server dependency by importing the db.json file directly using lowdb. See what I did here: https://github.com/asyncapi/glee/blob/master/examples/social-network/websocket-server/functions/onLikeDislike.js#L2.